### PR TITLE
Add code of conduct material, from covenant

### PR DESCRIPTION
**Pull request by YooSunYoung**
As @jl-wynen suggested, maybe we can add this CODE_OF_CONDUCT in our organization profile like this
and then have the same badge in `README.md` of all projects...?

### CODE REVIEW

File: profile/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (YooSunYoung) [inline profile/CODE_OF_CONDUCT.md]: I was not sure what contact to put in here though...

**Comment by jl-wynen** [comment]
Where would this show up on GH? If you add this file to a repo, there is a 'code of conduct' link on the right hand side in the same list as the license.

But regardless, I'd still put the file into every repo such people can find it even when they only have a clone.

**Comment by YooSunYoung** [comment]
> Where would this show up on GH? If you add this file to a repo, there is a 'code of conduct' link on the right hand side in the same list as the license.

It's our [organization page profile](https://github.com/scipp).
I thought that could be the main landing page of the future community...?
 
> But regardless, I'd still put the file into every repo such people can find it even when they only have a clone.

Yes, then we can have it in the template as well.

**Review by jl-wynen — APPROVED** [review]

File: profile/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (SimonHeybrock) [reply profile/CODE_OF_CONDUCT.md]: Same as in `copier_template`?

**Comment by YooSunYoung** [comment]
@jl-wynen I updated the contact same as copier template. Can you please review it and merge...? I can't merge it since I'm not the owner...!

File: profile/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-[INSERT CONTACT METHOD].
+scipp.ess.eu.
```
Comment (jl-wynen) [inline profile/CODE_OF_CONDUCT.md]:
```suggestion
scipp[at]ess.eu.
```
to show that this is an email address. We used @ in the scipp repo. `[at]` probably doesn't provide much protection nowadays but I'd still prefer it.

**Review by jl-wynen — CHANGES_REQUESTED** [review]

**Review by jl-wynen — APPROVED** [review]

**Pull request merged**